schutzbot/Jenkinsfile

```diff
@@ -11,6 +11,8 @@ pipeline {
         AWS_REGION = "us-east-2"
         AWS_BUCKET = "imagebuilder-jenkins-testing-use2"
         BUILD_CAUSE = detect_build_cause()
+        GCP_BUCKET = "osbuild-composer-testing"
```

Comment:
I wonder if we should move these two variables in `test/cases/api.sh`. IMHO, it would be more convenient to have the tests as much self-contained as possible and don't put additional login in Jenkins.

Reply:
Personally I don't think that we should. The reason I see is that the value is specific to the testing environment used for running the test case. This bucket is not auto-created by the test, therefore the test case script would be not usable out of the box if you would not have access to the same project that we use for CI. Also Bucket name must be unique globally (among all GCP projects), so user can not create it in their own project.